parent/pom.xml

```diff
@@ -242,6 +244,14 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
```

Comment:
Parent should not have any dependencies except dependencyManagement section, because it holds only settings, which will be used in maven modules where dependencies should be declared.

Reply:
Whoops, forgot to remove it before committing. Done.